docs/src/main/sphinx/appendix/from-hive.rst

```diff
@@ -110,6 +110,23 @@ Trino query::
     FROM tests
     CROSS JOIN UNNEST(scores) AS t (score);
 
+Use ANSI SQL syntax for date and time INTERVAL expressions
+----------------------------------------------------------
+
+Trino supports the ANSI SQL style ``INTERVAL`` expressions that differs from the implementation used in Hive.
+
+* The ``INTERVAL`` keyword is required and is not optional.
+* Date and time units must be singular. For example ``day`` and not ``days``.
+* Values must be quoted.
+
+Hive query::
+
+    SELECT cast('2000-08-19' as date) +  14 days;
```

Comment:
What Hive version are you using? Trino style query also works in Hive environment. Also, 
official example https://cwiki.apache.org/confluence/display/hive/languagemanual+types mentions `INTERVAL '1' DAY`.

Reply:
This was testing on Hive 2.x as would be deployed on Cloudera CDH 6. The Trino style query may work on Hive, but hive is much looser in what it will accept for date arithmetic. It's typically going from Hive to Trino that the problems occur.